Clinic/src/main/java/pl/infoshare/service/DoctorService.java

```diff
@@ -6,13 +6,13 @@
 import pl.infoshare.model.PersonDetails;
 import pl.infoshare.model.User;
 
+import static pl.infoshare.service.FileService.DOCTOR_PATH;
 
-public class DoctorService extends Login {
 
-    protected static final String DOCTOR_PATH = "Clinic/src/main/resources/doctor.json";
+public class DoctorService {
 
-    public static void addDoctor(User user) {
 
+    public static void addDoctor(User user) {
```

Comment:
skoro to jest zapis doktora to nazwę parametru można zmienić na doctor to samo w innych przypadkach jak pacjent
```suggestion
    public static void addDoctor(User doctor) {
```